search/retrieval-augmented-generation/examples/question_answering.ipynb

```diff
@@ -208,13 +210,18 @@
       },
       "outputs": [],
       "source": [
-        "PROJECT_ID = \"YOUR_PROJECT_ID\"  # @param {type:\"string\"}\n",
-        "SEARCH_ENGINE_ID = \"YOUR_SEARCH_ENGINE\"  # @param {type:\"string\"}\n",
+        "import os\n",
+        "\n",
+        "PROJECT_ID = \"python-docs-samples-tests\"  # @param {type:\"string\"}\n",
+        "DATA_STORE_LOCATION = \"eu\"  # @param {type:\"string\"}\n",
```

Comment:
Are non-global regions fully working now? I know you worked to update the Langchain support for non-global regions, and we also had some outages/issues with non-global regions.

Reply:
Yes, this will work once https://github.com/langchain-ai/langchain/pull/10513 goes through. 

Also, this made me realize I committed some actual project IDs... Will fix that.